index.js

```diff
@@ -40,15 +40,11 @@ function findRewardContainer (mutations) {
                 const wrap = node.getElementsByClassName('simplebar-content')[0]
```

Comment:
Recommending we stick with jQuery where applicable. Most of these lookups are redundant and we can get the reward-queue-view with one query

Reply:
Done. Due to Twitch's changes, we now search for and monitor `.reward-queue-body`.
```javascript
const queue = $(node).find('.reward-queue-body')[0]
```